src/features/rewards/tab/index.tsx

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import {
+  StyledWrapper,
+  StyledSlider,
+  StyledBullet,
+  StyledTextOne,
+  StyledTextTwo,
+  StyledSwitch
+} from './style'
+
+export interface Props {
+  id?: string
+  tabOneChecked?: boolean
+  tabOne?: string
+  tabTwo?: string
+  onSwitch?: (event: React.MouseEvent<HTMLDivElement>) => void
```

Comment:
Some suggestions for slightly better props naming:
- `onChange` as the event handler, and provide the index as the parameter instead of a MouseEvent
- `tabTitles` as a string array (throw an error if more than two are provided for now)
- `tabIndexSelected`

Reply:
done